feature-libs/product/configuration/src/textfield-configurator/facade/configurator-textfield.service.spec.ts

```diff
@@ -158,9 +158,9 @@ describe('ConfiguratorTextfieldService', () => {
     );
   });
 
-  it('should access the store with selector', () => {
+  it('should access the store when calling createConfiguration', () => {
```

Comment:
This works now, because you have only one rxjs operator inside of the pipe. There are two alternatives to test a selector:
- mock it - e.g. take a look at https://github.com/SAP/spartacus/blob/167801caddb2a1e4aab5dc81d83bbe5be4d1b008/projects/core/src/site-context/facade/currency.service.spec.ts#L88
- dispatch a Success action at the beginning of the test, that will pre-populate the store with necessary data